eslint/lib/rules/no-unblessed-new.js

```diff
@@ -29,8 +29,11 @@ module.exports = {
 
     create: function(context) {
         let allowed = {};
-        for (var i = 0; i < context.options.length; i++) {
-            allowed[context.options[i]] = 1;
+        const optionAllowed = context.options[0];
+        if (optionAllowed) {
+            for (var i = 0; i < optionAllowed.length; i++) {
+                allowed[optionAllowed[i]] = 1;
```

Comment:
curious why you set this to `1` instead of to `true`? 
I'm also curious why you used an object for this instead of an array (and used `allowed.includes(node.callee.name)` below).

Reply:
> 1 instead of to true

Perl habits die hard. Updated.

> object for this instead of an array

Also an old habit, since map-based lookups should always be fast whereas array-based lookups can be linear. But since that isn't a real concern here, I took a look at using `includes` instead and found that it isn't actually available - other default eslint rules that use it pull in lodash. Going to leave this as-is rather than add that dependency.